### DISCUSSION TITLE
relative path support for og_image for pages

### DISCUSSION THREAD

**Discussion by ammahmoudi** [alshedivat/al-folio#2612] [Q&A] [answered]
Hi. currently if you set an og_image for a page in relative format  like assets\...\xxxxx.png it wont work in link previews but it would work if you give the absloute path. is there a way to use relative path in here. i wanna avoid absloute path as the domain may change

**Comment by george-gca** [comment, marked as answer]
It works fine if you use a unix-style path instead of windows-style, for example `og_image: assets/img/prof_pic.png`.
